Example/StripeConnectExample/StripeConnect Example.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,704 @@
+// !$*UTF8*$!
```

Comment:
This can be done as a followup PR, but we also need to enable all the same locales as the SDK in the example app in order to test them.